### ISSUE TITLE
 app crashed and disabled accessibility

### ISSUE THREAD

**Issue by vishaljkk** [d-max/ussd#2]
The application crashes and disables accessibility and then runs again when installed but it does not work inside the app instead it displays dialog boxes for ussd above the app which wass not the intended thing for the app.
